pinot-controller/src/main/java/org/apache/pinot/controller/api/pojos/Instance.java

```diff
@@ -19,119 +19,127 @@
 package org.apache.pinot.controller.api.pojos;
```

Comment:
If possible, can we move this class to another package?

I think that this should be under `pinot-common`? For instance, `Tenant` is in `pinot-common/.../config`

Reply:
Good point. Moved